src/store/modules/HardwareStatus/ConcurrentMaintenanceStore.js

```diff
@@ -1,40 +1,27 @@
 import api from '@/store/api';
 import i18n from '@/i18n';
+import { defineStore } from 'pinia';
 
-const ConcurrentMaintenanceStore = {
-  namespaced: true,
-  state: {
-    readyToRemove: false,
+export const ConcurrentMaintenanceStore = defineStore('ConcurrentMaintenance', {
```

Comment:
Let us keep this defineStore('**ConcurrentMaintenance**' as "**concurrentMaintenance**". C -> lowercase

Reply:
updated